instrumentation/spring/spring-boot-autoconfigure/src/test/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/otlp/OtlpLogExporterAutoConfigurationTest.java

```diff
@@ -64,39 +61,4 @@ void otlpLogsDisabled() {
         .withPropertyValues("otel.logs.exporter=none")
         .run(context -> assertThat(context.containsBean("otelOtlpLogRecordExporter")).isFalse());
   }
-
-  @Test
-  void otlpHttpUsedByDefault() {
```

Comment:
a lot of the removed tests seem useful, could you elaborate why you chose to remove them?

Reply:
Now that we're re-using the SDK logic, we can (and I think should) rely of the tests that are in the SDK.

`OtlpExporterPropertiesTest` is the new test that specifically makes sure properties of all types are translated correctly.

I've kept the tests for disabling the beans, because that part is not delegated to the SDK.